specification/machinelearningcompute/resource-manager/Microsoft.MachineLearningCompute/2017-08-01-preview/machineLearningCompute.json

```diff
@@ -882,11 +892,11 @@
         },
         "cert": {
           "type": "string",
-          "description": "The SSL cert data in PEM format encoded as base64 string"
```

Comment:
No functional changes here. The cert/key were never being passed as base64 string so I'm fixing the comment.